manifests/README.md

```diff
@@ -0,0 +1,78 @@
+# Auto-heal Example Manifests
+
+This directory contains examples of manifests files that can be used to deploy
+the auto-heal service to an _OpenShift_ or _Kubernetes_ cluster.
+
+## Customization
+
+These manifests deploy the auto-heal service to a new `openshift-monitoring`
+namespace, which will be created by the `autoheal-namespace.yml` file. If you
+wish to use a different namespace then you will have to change all the files to
+reflect the new one. That can be done with the following command:
+
+```
+$ sed -i 's/openshift-monitoring/my-namespace/g' *.yml
+```
+
+The configuration of the auto-heal service is stored in the configuration map
+that is created by the `autoheal-config-map.yml` manifest file. You will
+probably want to change that file, specially the `awx` section, to use your own
+AWX or Ansible Tower service.
+
+The credentials and the CA certificates used to connect to the AWX or
+AnsibleTower server are stored in the secrets created by the
+`awx-credentials-secret.yml` and `awx-tls-secret.yml` files, so you will wanto
+to change those as well. For example, to use the user name `autoheal` and the
+password `redhat123` do the following:
+
+```
+$ echo -n 'autoheal' | base64
+YXV0b2hlYWw=
+$ echo -n 'redhat123' | base64
+cmVkaGF0MTIz
+```
+
+Then modify the `awx-credentials-secret.yml` file so that it looks like this:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: openshift-monitoring
+  name: awx-credentials
+data:
+  username: YXV0b2hlYWw=
+  password: cmVkaGF0MTIz
+```
+
+To use the contents of the file `ca.crt` as the CA certificates do the following:
+
+```yaml
+$ base64 ca.crt
+LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUMvVENDQWVXZ0F3SUJBZ0lKQUxNRXB6OWxa
+...
+```
+
+Then modify the `awx-tls-secret.yml` file to that it contains the output of the
+above command, should look like this:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: openshift-monitoring
+  name: awx-tls
+data:
+  ca.crt: |-
+    LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUMvVENDQWVXZ0F3SUJBZ0lKQUxNRXB6OWxa
+    ...
+```
+
+## Deployment
+
+Once you have customized the manifests as described in the above section, you
+can just run all of them using the `oc` or `kubectl` tools:
+
+```
+$ oc create -f *.yml
```

Comment:
maybe `oc apply` is better? it allows updating the configuration

Reply:
also when you run with *.yml you depend on the alphabetical order of the files. So `autoheal-config-map.yml` will be created before the `autoheal-namespace.yml` is created, and it might fail...
Maybe we should add all the manifests to a single openshift template to eliminate dependencies?